packages/core/core/src/utils/context/index.ts

```diff
@@ -4,29 +4,27 @@ interface ContextConfiguration {
   useVSFContext: () => Context;
 }
 
+interface ApplyingContextHooks {
+  before: (args: any[]) => any[];
+  after: (response: any) => any;
+}
+
 let useVSFContext = () => ({}) as Context;
 
 const configureContext = (config: ContextConfiguration) => {
   useVSFContext = config.useVSFContext || useVSFContext;
 };
 
-const applyContextToApi = (api, context, extensions = []) =>
+const NOP = (x) => x;
```

Comment:
Its worth describing why we need no-op here and what this fn does as its hard to understand from code